athenahealth/lab_results.go

```diff
@@ -0,0 +1,303 @@
+		}
+		if opts.TieToOrderID != nil {
+			form.AddInt("tietoorderid", *opts.TieToOrderID)
+		}
+	}
+
+	out := &addLabResultDocumentResponse{}
+
+	_, err := h.PostFormReader(ctx, fmt.Sprintf("patients/%s/documents/labresult", patientID), form, out)
+	if err != nil {
+		return 0, err
+	}
+
+	if !out.Success {
+		return 0, errors.New(out.ErrorMessage)
+	}
+
+	return out.LabResultID, nil
+}
+
+type ListChangedLabResultsOptions struct {
+	ShowPortalOnly             *bool     `json:"showportalonly"`
+	LeaveUnprocessed           *bool     `json:"leaveunprocessed"`
+	ShowProcessedEndDateTime   time.Time `json:"showprocessedenddatetime"`
+	ShowProcessedStartDateTime time.Time `json:"showprocessedstartdatetime"`
+
+	Pagination *PaginationOptions
+}
+
+type ListChangedLabResultsChangedLabResult struct {
+	AppointmentID        int    `json:"appointmentid"`
+	AssignedTo           string `json:"assignedto"`
+	CreatedDate          string `json:"createddate"`
+	CreatedDateTime      string `json:"createddatetime"`
+	CreatedUser          string `json:"createduser"`
+	DeletedDateTime      string `json:"deleteddatetime"`
+	DepartmentID         string `json:"departmentid"`
+	Description          string `json:"description"`
+	DocumentClass        string `json:"documentclass"`
+	DocumentRoute        string `json:"documentroute"`
+	DocumentSource       string `json:"documentsource"`
+	DocumentSubclass     string `json:"documentsubclass"`
+	EncounterDate        string `json:"encounterdate"`
+	EncounterID          string `json:"encounterid"`
+	ExternalNoteOnly     string `json:"externalnoteonly"`
+	FacilityID           int    `json:"facilityid"`
+	FileExtension        string `json:"fileextension"`
+	InternalNote         string `json:"internalnote"`
+	Interpretation       string `json:"interpretation"`
+	IsConfidential       string `json:"isconfidential"`
+	IsReviewedByProvider string `json:"isreviewedbyprovider"`
+	LabResultID          int    `json:"labresultid"`
+	LabResultLOINC       string `json:"labresultloinc"`
+	LastModifiedDate     string `json:"lastmodifieddate"`
+	LastModifiedDateTime string `json:"lastmodifieddatetime"`
+	NoteFromLab          string `json:"notefromlab"`
+	ObservationDate      string `json:"observationdate"`
+	ObservationDateTime  string `json:"observationdatetime"`
+	OrderType            string `json:"ordertype"`
+	OriginalFilename     string `json:"originalfilename"`
+	PatientID            int    `json:"patientid"`
+	PatientNote          string `json:"patientnote"`
+	Priority             string `json:"priority"`
+	ProviderID           int    `json:"providerid"`
+	ReportStatus         string `json:"reportstatus"`
+	ResultStatus         string `json:"resultstatus"`
+	Status               string `json:"status"`
+	Subject              string `json:"subject"`
+	TieToOrderID         int    `json:"tietoorderid"`
+}
+
+type listChangedLabResultsResponse struct {
+	LabResults []*ListChangedLabResultsChangedLabResult `json:"labresults"`
```

Comment:
Do we want to call this key `ChangedLabResults` to make a distinction between the standard `LabResult` type?

Reply:
Yeah, I was thinking perhaps this should be consistent with athena's response but that makes more sense